sal_interfaces/MTAOS/MTAOS_Commands.xml

```diff
@@ -3,244 +3,159 @@
+      <EFDB_Name>config</EFDB_Name>
+      <Description>Yaml configuration string with additional parameters to be passed to ISR and other pre-process tasks. If options are invalid command will be rejected. If not given, use the standard configuration.</Description>
       <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAOS</Subsystem>
+    <EFDB_Topic>MTAOS_command_runWEP</EFDB_Topic>
+    <Description>Process wavefront data, either full focal plane mode (Comcam and LSSTCam) or wavefront sensors. If only visitId is provided it assumes wavefront sensor mode. If visitId and extraId is provided it assumes full focal plane mode. MTAOS will determine if it is ComCam or LSSTCam based on the input images. If useOCPS=True, processing is done remotely using the OCPS CSC. For LSSTCam full array mode only OCPS processing is supported. Before running this command it is advisable to run selectSources and preProcess commands on the input image(s), otherwise MTAOS will have to run those processes before running wep.</Description>
     <item>
-      <EFDB_Name>extraDirectoryPath</EFDB_Name>
-      <Description>Absolute directory path to extra-focal wavefront data collection.</Description>
-      <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
+      <EFDB_Name>visitId</EFDB_Name>
+      <Description>Image visit id number. This is a unique identifier for each image and consists of the date of the observation plus an integer iterator, e.g. 2020103000040 for the 40th image taken in October, 30, 2020.</Description>
+      <IDL_Type>long</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>fieldRA</EFDB_Name>
-      <Description>Right ascension of telescope boresight.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>degree</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>fieldDEC</EFDB_Name>
-      <Description>Declination of telescope boresight.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>degree</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>filter</EFDB_Name>
-      <Description>Active filter type. Use 1-6 for u, g, r, i, z, and y. Use 7 for reference.</Description>
-      <IDL_Type>int</IDL_Type>
+      <EFDB_Name>extraId</EFDB_Name>
+      <Description>Extra focal image visit id number (optional). Same as visitId, if different than zero, the command assumes this is an Intra/Focal sequence, otherwise it assumes Corner Wavefront Sensor.</Description>
+      <IDL_Type>long</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>cameraRotation</EFDB_Name>
-      <Description>Camera rotation angle.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>degree</Units>
+      <EFDB_Name>useOCPS</EFDB_Name>
```

Comment:
What will happen if the user put `useOCPS=true` in `MTAOS_command_preProcess()` and put the value of `false` in this command (`MTAOS_command_runWEP()`)? And vice versa?

Reply:
For the LSSTCam full array mode, `preProcess` and `runWEP` will only be supported for `useOCPS=True`. So if you send either of them with `useOCPS=False`, the command is rejected. Then, if you send `runWEP` without a previous `preProcess`, the pre-processing will be done before WEP. We should probably send a warning message in this case. 

For ComCam full array I think the command should run either way. In this case, doing `preProcess` with `useOCPS=False` and `runWEP` with `useOCPS=True` will be the same as if just executing `runWEP` with `useOCPS=True`, and vice-versa. In either case, we should probably send a warning message and it could be a special message saying that preProcess was run with a different `useOCPS` setup. 

I am weary to *forcibly require* `preProcess` to run with the same `useOCPS` because there are cases where we may simply not do `preProcess` and just want to execute `runWEP` and let it deal with all the steps. 

For LSST corner wavefront sensor there's no `preProcess` step so, that's it.

Reply:
I added some information on the xml about this.

Reply:
I do not see where is the added information. BTW, it is hard to trace the update if you continue to use the force-push to replace the latest commit. At each time, the reviewer will need to re-read the commit to look for the updated part. Thanks!

Reply:
It's in the description, one line 105

```
<Description>Use OCS-Controlled Pipeline System (OCPS) to process data remotely? Ideally this should match the value of a previous preProcess command. If runWEP and preProcess run with different useOCPS setups, runWEP will run normally but will take longer as it will have to repeat the steps of preProcess. A warning message will inform the users of these conditions.</Description>
```

Reply:
Got it. Thanks!